src/libs/Network/MainQueue.ts

```diff
@@ -1,7 +1,7 @@
-import * as Request from '@libs/Request';
+import {processWithMiddleware} from '@libs/Request';
 import type OnyxRequest from '@src/types/onyx/Request';
-import * as NetworkStore from './NetworkStore';
-import * as SequentialQueue from './SequentialQueue';
+import {isAuthenticating, isOffline} from './NetworkStore';
+import {isRunning as sequentialQueueIsRunning} from './SequentialQueue';
```

Comment:
NAB - for consistency with naming boolean variables and functions, we could have named this `isSequentialQueueRunning` instead